src/main/java/frc/robot/DriveSystem.java

```diff
@@ -5,56 +5,42 @@
 public class DriveSystem {
     private double _driveCmd;
     private double _turnCmd;
-
+    //Initialization of variables
     private Talon _motorFL;
     private Talon _motorFR;
     private Talon _motorRL;
     private Talon _motorRR;
 
-    /**
-     * Default constructor
-     * 
-     * Instantiates the motor Talon objects with their correct channels and inversion state based
-     * on constants contained in the RobotConstants class, e.g. -
-     * 
-     *      MOTOR_CHANNEL_FL, MOTOR_CHANNEL_FR, ...
-     *      MOTOR_INVERTED_FL, MOTOR_INVERTED_FR, ...
-     * 
-     * Also sets the commands to 0.0 by default.
-     */
+    //MOTOR_CHANNEL_FL, MOTOR_CHANNEL_FR, ...
+    //MOTOR_INVERTED_FL, MOTOR_INVERTED_FR, ...
     public DriveSystem() {
+        this._motorFL = new Talon(RobotConstants.MOTOR_CHANNEL_FL);
+        this._motorFR = new Talon(RobotConstants.MOTOR_CHANNEL_FR);
+        this._motorRL = new Talon(RobotConstants.MOTOR_INVERTED_FL);
+        this._motorRR = new Talon(RobotConstants.MOTOR_INVERTED_RR);
 
+        this.reset();
     }
-
-    /**
-     * Sets the commands to 0.0 and makes sure all motors are also set to 0.0.
-     */
     public void reset() {
-
+        this.setCommands(0.0, 0.0);
     }
 
-    /**
-     * Updates the drive and turn commands used to command the motors based on an external input.
-     * 
-     * @param drive desired forward/backward command for the robot in the range [-1.0, 1.0]
-     * @param turn desired left/right command for the robot in the range [-1.0, 1.0]
-     */
-    public void setCommands(double drive, double turn) {
+     //@param drive desired forward/backward command for the robot in the range [-1.0, 1.0]
+     //@param turn desired left/right command for the robot in the range [-1.0, 1.0]
 
+    public void setCommands(double driveCmd, double turnCmd) {
+        this._driveCmd = driveCmd;
+        this._turnCmd = turnCmd;
     }
-
-    /**
-     * Commands the motors based on the most recently provided drive and turn commands. The
-     * output for each motor on the robot are as follows:
-     * 
-     *      left side: (drive - turn)
-     *      right side: (drive + turn)
-     * 
-     *      s.t. - all motor commands are in the range [-1.0, 1.0]
-     *             drive is positive in the forward direction
-     *             turn is positive in the CCW direction
-     */
+    //left is drive + turn
+    //right is drive - turn
     public void update() {
-        // It is encouraged that you re-derive the equations to convince yourself that they are true.
+        double leftTrn = MaxCalc.limit(this._driveCmd + this._turnCmd,-1.0, 1.0);
+        double rightTrn = MaxCalc.limit(this._driveCmd - this._turnCmd,-1.0, 1.0);
+
+        this._motorFL.set(leftTrn);
+        this._motorFR.set(leftTrn);
```

Comment:
Interesting that you're setting the front right motor to the command you calculated for the left side.